.tool-versions

```diff
@@ -1,2 +1,3 @@
 elixir 1.10.3-otp-23
 erlang 23.0.2
+python 3.9.1
```

Comment:
Can we bump this to `3.9.2` since that's the latest version, and what `gtfs_creator` is using?

Reply:
Given that a month has passed since this comment, and that `3.9.3` contained some security fixes, I'm going to jump straight to `3.9.4` if that's alright. https://www.python.org/downloads/release/python-394/

Reply:
https://www.python.org/downloads/release/python-395/